state/src/main/scala/cats/state/State.scala

```diff
@@ -78,6 +78,12 @@ final class StateT[F[_], S, A](val runF: F[S => F[(S, A)]]) {
   def modify(f: S => S)(implicit F: Monad[F]): StateT[F, S, A] =
     transform((s, a) => (f(s), a))
 
+  /**
+   * Extract a value from the input state, without modifying the state.
```

Comment:
I also thought about changing that. I decided that using a synonym in the doc instead of a somewhat cyclical definition of "inspect" might be more helpful, but I'm happy to change.